app/lib/till.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'order'
+require_relative 'receipt'
+require_relative 'payment'
+require_relative 'menu'
+
+
+class Till
+
+  include Menu
```

Comment:
This should not be a module - modules are used when you want to add extra attributes to an existing object which may be shared with other objects. But in this case I would argue that Menu is it's own object, which rather than including in Till you could just call from within Till:

```
class Till
 attr_reader :menu

  def initialize
    @menu = Menu.new
  end

  def items
    menu.items
  end
end
```

A simple way for me of minimising using modules is to always end my modules with -able (e.g. `runnable` and `walkable` could be included in a `Human` object), if I can't think of an `-able` word that fits it's probably not suitable for a module.

But in general anyway we prefer to use objects instead wherever possible and *delegate* to these objects like I've done above.